test/tests/api/emails.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+describe('/POST emails', ()=> {
+  it('should save message to DB', (done)=> {
+    let message = {from_email: 'ssing128@gmail', to_email: 'ssing128@gmail', body: 'i love you'};
+    request
+      .post('/emails')
+      .send(message)
+      .expect(200)
+      .end((err,res)=> {
+        if(err) return done(err);
+        res.body.should.be.a('object');
```

Comment:
Your test needs to actually test that the body looks like what you think it shoud:
Check to make sure it has an `id` maybe, test something! Check to see if it has `body === message.body`. Etc.

Reply:
Got it.